runhouse/resources/hardware/cluster.py

```diff
@@ -967,6 +970,11 @@ def restart_server(
             )
             + " --from-python"
             + f" --log-level {logs_level}"
+            + (
+                " --enable-telemetry-collection"
+                if enable_telemetry_collection
+                else " --no-enable-telemetry-collection"
```

Comment:
we shouldn't have to explicitly set it to not being enabled, if the `--enable-telemetry-collection` flag is provided that should be enough

Reply:
yeah but if the user does not provide `enable_telemetry_collection`, we set it by default to `true`. Therefore, we need to check if the user provided `enable_telemetry_collection = false `when calling cluster.restart_server(), and than restart it with no telemetry collection. Or do I miss something here?